security/encrypt-cluster-pod-traffic.md

```diff
@@ -36,7 +36,92 @@ This how-to guide uses the following {{site.prodname}} features:
 
 1. Install WireGuard on cluster nodes using {% include open-new-window.html text='instructions for your operating system' url='https://www.wireguard.com/install/' %}. Note that you may need to reboot your nodes after installing WireGuard to make the kernel modules available on your system.
 
-    > **Note**: Nodes that do not support WireGuard will not be secured by WireGuard tunnels, even if traffic running on the node to and from the pods goes to nodes that do support WireGuard.
+   Check the following if your operating system is not listed on the WireGuard installation page.
```

Comment:
Use the following instructions for operating systems not listed on the WireGuard installation page.